projects/pillow/project.yaml

```diff
@@ -1,13 +0,0 @@
-homepage: "https://python-pillow.org/"
```

Comment:
Why remove pillow?

Reply:
It has never functioned properly. It's complex to get this working. I've never requested or received a reward for Pillow. I don't want to fix it now. Maybe another time.